Solutions/Windows Security Events/Analytic Rules/NRT_base64_encoded_pefile.yaml

```diff
@@ -20,9 +20,9 @@ relevantTechniques:
   - T1140
 query: |
  SecurityEvent
-  | where EventID==4688
+  | where EventID == 4688
   | where isnotempty(CommandLine)
-  | where CommandLine contains "TVqQAAMAAAAEAAA"
+  | where CommandLine has "TVqQAAMAAAAEAAA"
 entityMappings:
   - entityType: Account
     fieldMappings:
```

Comment:
We should update account entity mapping to strong identifiers

Reply:
Done